draft-ietf-dnssd-privreq.xml

```diff
@@ -44,7 +44,10 @@
    'http://xml2rfc.ietf.org/public/rfc/bibxml/reference.RFC.8235.xml'>
 <!ENTITY rfc8236 PUBLIC ''
    'http://xml2rfc.ietf.org/public/rfc/bibxml/reference.RFC.8236.xml'>
-
+<!ENTITY I-D.ietf-dnssd-pairing PUBLIC ''
+   "http://xml2rfc.ietf.org/public/rfc/bibxml3/reference.I-D.ietf-dnssd-pairing.xml"> 
+<!ENTITY I-D.huitema-dnssd-privacy PUBLIC ''
+   "http://xml2rfc.ietf.org/public/rfc/bibxml3/reference.I-D.huitema-dnssd-privacy.xml"> 
 <!ENTITY I-D.ietf-intarea-hostname-practice PUBLIC ''
```

Comment:
Reference should be I-D.ietf-dnssd-privacy.

Reply:
Will fix.

Reply:
Done.